src/core_plugins/kbn_vislib_vis_types/public/controls/heatmap_options.html

```diff
@@ -163,21 +191,29 @@
         </table>
 
         <div class="hintbox" ng-show="!editorState.params.colorsRange.length">
-          <p>
-            <span class="kuiIcon fa-danger text-danger"></span>
-            <strong>Required:</strong> You must specify at least one range.
-          </p>
+          <p
+            i18n-id="kbnVislibVisTypes.controls.heatmapOptions.specifiedRangeNumberWarningMessage"
+            i18n-default-message="{icon} {required} You must specify at least one range."
+            i18n-values="{
+              icon: '<span class=\'kuiIcon fa-danger text-danger\'></span>',
```

Comment:
Please use backtick strings to prevent escaping single quotes.

Reply:
There is error if we will use backtick.
![image](https://user-images.githubusercontent.com/23155410/46946223-c0456f00-d07f-11e8-91f5-6b8c2de4d0a3.png)

Reply:
Angular does not allow  use backtick strings in template